pkg/config/assets/service_config.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "eth",
+    "id": "some-id",
+    "description": "some-desc",
+    "bindable": true,
+    "tags": [
+      "eth",
+      "geth"
+    ],
+    "metadata": {
+      "displayName": "some-name",
+      "longDescription": "some-long-desc",
+      "providerDisplayName": "some-provider-display-name"
+    },
+    "dashboard_client": {
+      "id": "some-client-id",
+      "secret": "some-secret"
+    },
+    "plans": [
+       {
+          "id": "some-plan-id",
+          "name": "free",
+          "description": "free-trial",
+          "free": true,
+          "metadata": {
+             "displayName": "service-plan-metadata",
+             "container": {
+               "backend": "docker",
+               "image": "some-image"
```

Comment:
Do we want to have this metadata on the service config or something the broker knows?
As in I think this is information for the broker not necessarily for the catalog?

Reply:
I agree in that it probably doesnt need to be exposed as part of the catalog. but we still want to tie it to a service definition config somehow in such a way that a given variation of the service can use a different docker repo/image.

Reply:
yea i agree with that, then we should probably test that this data does not show up in the service definition.

Reply:
SGTM